CHANGES.txt

```diff
@@ -1,4 +1,7 @@
+2.5.0 (2024-04-04)
 
+Added:
+- Support Subscriptions API hosting block for Sentinel Hub in the CLI (#1029).
 
 Added:
  - Support for field_boundaries_sentinel_2_p1m in Subscriptions API (#1026)
```

Comment:
I think these could be bullet points under the same `Added:` list because both MRs are being tagged as v2.5.0?